infra/modules/providers/azure/app-insights/main.tf

```diff
@@ -0,0 +1,17 @@
+module "azure-provider" {
+    source = "../provider"
+}
+
+resource "azurerm_resource_group" "appinsights" {
+  name     = "${var.resource_group_name}"
+  location = "${var.resource_group_location}"
+  tags     = "${var.resource_tags}"
+}
+
+resource "azurerm_application_insights" "appinsights" {
```

Comment:
can you include an example for how one would configure an AI instance to 1) a designated app service 2) to api manager.

Reply:
These are all the available options in app insights for azure in terraform. I would suppose, it can be done using a script after deployment is complete but I don't have any examples of how its done.